lib/models/requestInput.js

```diff
@@ -128,6 +131,13 @@ class RequestInput {
       }
     });
 
+    // initialize `_headers` property after the population of `this.args` attribute
```

Comment:
we need to keep a comment here to explain why the following line is not higher in the code.
... but I'm not really satisfied with the wording, so if anyone has a better suggestion...